List of changes on the addUsersLk branch

MrNikaMilon opened the pull request:
Implemented the following functionality -
- Missing method in the Menu class for User interaction with his personal account.
- I also had to add a new method for returning Login to the User functionality

MrNikaMilon committed:
List of changes on the addUsersLk branch:
Implemented the following functionality -
- Missing method in the Menu class for User interaction with his personal account.
- I also had to add a new method for returning Login to the User functionality

MaksMusic commented:
private void lkUser(UserAccount userAccount){

MaksMusic commented:
fix the method private void lkUser(UserAccount userAccount) ,case 2  in class Menu.
At the moment it does not change the user id
1 you don’t need to accept the user ID at all to change his password
2 in this method you pass the id to the method userService.retrunUserLogi() that returns the user’s login and does not change the user’s id

MrNikaMilon committed:
List of changes on the addUsersLk branch:
Implemented the following functionality -
- Missing method in the Menu class for User interaction with his personal account.
- I also had to add a new method for returning Login to the User functionality

List of changes dated 08/05/2024
Fixed the work of my methods, changed the input data - now we get login and use it to look for identifier and work with the record.

MrNikaMilon commented:
All changes have been made.
